modules/social_features/social_embed/social_embed.routing.yml

```diff
@@ -0,0 +1,23 @@
+social_embed.consent:
+  path: '/api/opensocial/social-embed/consent'
+  defaults:
+    _controller: '\Drupal\social_embed\Controller\EmbedController::checkConsent'
+    _title: 'Embed consent'
+  requirements:
+    _access: 'TRUE'
+
+social_embed.generate:
+  path: '/api/opensocial/social-embed/generate'
+  defaults:
+    _controller: '\Drupal\social_embed\Controller\EmbedController::generateEmbed'
+    _title: 'Generate embed'
+  requirements:
+    _access: 'TRUE'
```

Comment:
I think this should be only available for LU, so we don't have an AN DDOS opportunity